### DISCUSSION TITLE
Integrations with other 3rd party Extensions (Official)

### DISCUSSION THREAD

**Discussion by KEZEO** [JoomCoder-com/JoomCCK#30] [Ideas]
Hello 
an integration with others joomboost projects will be amazing

**Comment by andiamine** [comment]
Yes definitely we are going to do that. JoomCCK will be integrated with many extensions , here list of most wanted
- JoomGrabber Adapter for JoomCCK
- JoomRSS content source for JoomCCK
- JoomForms submission mapping with JoomCCK
- JoomPMS integration with JoomCCK
- JoomSMS integration with JoomCCK

**Comment by Giorgi625** [comment]
Hello @KEZEO , could you post about this https://github.com/JoomBoost/JoomCCK/discussions/34 on mintjoomla ?  I was blocked by asmuz. maybe someone will be able to join and help to speed up development of JoomCCK.

**Comment by andiamine** [comment]
Quick note, also will be an integration with JoomCoder products.

**Comment by andiamine** [comment]
More integrations that will be done with JoomBoost products:
- JoomCRM plugin to submit form as lead and in JoomCCK
- JoomProject custom field that let you choose project and display summary of it.
- Easy Services Booking custom field that let you choose appointement and display booking button.

**Reply by JEROMELECORSE** [reply]
> More integrations that will be done with JoomBoost products:
> 
> * JoomCRM plugin to submit form as lead and in JoomCCK
> * JoomProject custom field that let you choose project and display summary of it.
> * Easy Services Booking custom field that let you choose appointement and display booking button.

joomcrm important because i have a compagny

**Comment by Giorgi625** [comment]
that will be good, also I see coming soon Joomcart.
I did not really liked how J2store was integrated into Cobalt so hope we will have some better cart integration.
Especially will be cool if there will be multivendor option.

**Reply by andiamine** [reply]
Yes we are working on JoomCart and JoomERP as side projects.

JoomCart will be based on OpenCart, like MijoShop if you remember this extension
JoomERP will be based on Dolibarr , is a very huge extension and will have tons of features 

JoomCoder will have enterprise level extensions
Also forgot JoomSubscription will be based on Emerald , and we will try to add recurring on it in future.

**Comment by Giorgi625** [comment]
Yes I liked Mijoshop, unlike many other cart extensions on Joomla.
Wanted to ask about Emerald and forgot, thats good news. JoomSubscription and JoomCCK together will be most powerful tool for many tasks.

**Comment by andiamine** [comment]
Yes 100% will be still integrated with JoomCCK
